practice-one/src/components/Form/index.tsx

```diff
@@ -0,0 +1,278 @@
+        setNameError("");
+        setEmailError("");
+        setPhoneError("");
+        setPasswordError("");
+        setImageError("");
+    };
+
+    return (
+        <form className="add-student-form" onSubmit={handleSubmit}>
+            <div className="form-header">
+                <h2 className="heading-form">Add Students</h2>
+                <div className="link-group">
+                    <a href="#" className="link-group-item">Manually</a>
+                    <a href="#" className="link-group-item">Import CSV</a>
+                </div>
+            </div>
+            <div className="form-body">
+                <div className="form-body-item">
+                    <div className="student-name item">
+                        <p className="item-title">Name</p>
+                        <Input
+                            className="input-default"
+                            type="text"
+                            value={name}
+                            name="name"
+                            onChange={handleNameChange}
+                        />
+                        {nameError && <span className="error-message">{nameError}</span>}
+                    </div>
+                    <div className="class-student item">
+                        <select
+                            name="classes"
+                            className="select-item"
+                            value={classes}
+                            onChange={handleClassesChange}>
+                            <option value="default">Class</option>
+                            <option value="SS1">SS1</option>
+                            <option value="SS2">SS2</option>
+                            <option value="SS3">SS3</option>
+                            <option value="SS4">SS4</option>
+                            <option value="SS5">SS5</option>
+                            <option value="SS6">SS6</option>
+                        </select>
+                        {classError && <span className="error-message">{classError}</span>}
+                    </div>
+                    <div className="gender-student item">
+                        <select name="gender"
+                            className="select-item"
+                            value={gender}
+                            onChange={handleGenderChange}>
+                            <option value="default">Gender</option>
+                            <option value="Male">Male</option>
+                            <option value="Female">Female</option>
+                        </select>
+                        {genderError && <span className="error-message">{genderError}</span>}
+                    </div>
+                </div>
+                <div className="form-body-item">
+                    <div className="studen-email item">
+                        <p className="item-title">Email</p>
+                        <Input
+                            className="input-primary"
```

Comment:
with `input-primary` class please use default value.
or you can use variant style
example:

input: {
primary: {
  color: 'red',
....
},
secondary: {...
}
}